### PR TITLE
8.0 task work invoice

### PR DESCRIPTION
**Pull request by bguillot**
Hide the create related task action if user has no rights
Add task work view 
Add wizard to set an invoice number on task work

Improvments can be made it's not perfect !

### CODE REVIEW

**Comment by florian-dacosta** [comment]
Seems good to me. We will see after using a little.
:+1:

File: project_ak/wizard/invoice_work.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
```
Comment (bealdav) [inline project_ak/wizard/invoice_work.py:5]: on pourrait abréger l'entete ?

**Comment by bealdav** [comment]
LGTM

File: project_ak/wizard/invoice_work.py (hunk 1 above)
Comment (bguillot) [inline project_ak/wizard/invoice_work.py:5]: C'est fait !

**Pull request merged**